extension/js/common/core/pgp-msg.ts

```diff
@@ -211,6 +221,19 @@ export class PgpMsg {
   }
 
   public static encrypt: PgpMsgMethod.Encrypt = async ({ pubkeys, signingPrv, pwd, data, filename, armor, date }) => {
+    // slice(1) filters first key that is always own OpenPGP key
+    // in result if X.509 encryption is selected the e-mail author
+    // cannot decrypt e-mails that they have sent
+    const otherKeys = pubkeys.slice(1);
```

Comment:
I still don't like this, I would not rely on key order as indicator of which key is mine. Will refactor